src/pybind/tests/test_feat.py

```diff
@@ -19,17 +20,17 @@ class TestFeat(unittest.TestCase):
 
     def test_mfcc(self):
         mfcc = feat.Mfcc(feat.MfccOptions())
-        reader = SequentialWaveReader("ark:tests/wav.ark")
+        reader = SequentialWaveReader('ark:tests/wav.ark')
         # gold set is feature extracted using featbin/compute-mfcc-feats
-        gold_reader = SequentialMatrixReader("ark:tests/feat.ark")
+        gold_reader = SequentialMatrixReader('ark:tests/feat.ark')
         for key, value in reader:
             print("Validate utterance: {}".format(key))
```

Comment:
still a double quote here.

Even though, double quotes and single quotes are both fine in Python for strings,
but for consistency, I think only one kind of quotes should be used and used consistently
throughout all python scripts.

Reply:
I know it's difficult to follow this style. When I violate it, please correct me.